### ISSUE TITLE
Nagstamon 2.0 leave behind 

### ISSUE THREAD

**Issue by Cola20150301** [HenriWahl/Nagstamon#306]
mac os OS 10.11.6 ,when i open Nagstamon 2.0,once it run ok,but when i quit it ,it leave behind and can't open .maybe i use chinese for monitor server description

**Comment by HenriWahl** [comment]
Hi,
as Python3 and Qt5 fully support Unicode I do not think chinese is a problem. But you can try with another to check it out.
I do not fully understand how to reproduce this problem. What do you mean with "quit it ,it leave behind and can't open"? You can't restart it after quitting it?

**Comment by Cola20150301** [comment]
yes,now it can't open ,when i click Nagstamon  icon,it's gone away...

**Comment by HenriWahl** [comment]
Please check in process monitor (https://support.apple.com/en-us/HT201464) if there is still some piece of Nagstamon running.
You might also try to delete your Nagstamon settings - remove folder /Users/<your-user-name>/.nagstamon .

**Comment by Cola20150301** [comment]
it's ok when i delete the config file '.nagstamon/servers/server_测试.conf'.so ...thank u

**Comment by HenriWahl** [comment]
OK so you were right - it depends on the chinese characters. I tried to reproduce this under Linux with the name 测试 you posted her - and the same happens, Nagstamon dies. Might be a problem of Qt5? I do not know. If you like we can keep this issue open but I do not actually know how to fix it.

**Comment by HenriWahl** [comment]
Please check latest testing version from https://nagstamon.ifw-dresden.de/download/#testing.
